diff --git a/src/fvdb/detail/ops/gsplat/RasterizeScreenSpaceGaussiansBackward.cu b/src/fvdb/detail/ops/gsplat/RasterizeScreenSpaceGaussiansBackward.cu
index 47af543..3dc3187 100644
--- a/src/fvdb/detail/ops/gsplat/RasterizeScreenSpaceGaussiansBackward.cu
+++ b/src/fvdb/detail/ops/gsplat/RasterizeScreenSpaceGaussiansBackward.cu
@@ -1356,7 +1356,8 @@ callRasterizeBackwardPrivateUse1(
             int64_t elementOffset, elementCount;
             std::tie(elementOffset, elementCount) = deviceChunk(outTensor.numel(), deviceId);
             if (elementCount > 0) {
-                prefetchPointers.emplace_back(outTensor.data_ptr<ScalarType>() + elementOffset);
+                prefetchPointers.emplace_back(outTensor.template data_ptr<ScalarType>() +
+                                              elementOffset);
                 prefetchSizes.emplace_back(elementCount * sizeof(ScalarType));
             }
         }
diff --git a/src/fvdb/detail/ops/gsplat/RasterizeWorldSpaceGaussiansBackward.cu b/src/fvdb/detail/ops/gsplat/RasterizeWorldSpaceGaussiansBackward.cu
index 24f4a4a..3c19aa4 100644
--- a/src/fvdb/detail/ops/gsplat/RasterizeWorldSpaceGaussiansBackward.cu
+++ b/src/fvdb/detail/ops/gsplat/RasterizeWorldSpaceGaussiansBackward.cu
@@ -693,7 +693,7 @@ launchBackwardPrivateUse1(const torch::Tensor &means,
         for (const auto &outTensor: outTensors) {
             const auto [elementOffset, elementCount] = deviceChunk(outTensor.numel(), deviceId);
             if (elementCount > 0) {
-                prefetchPointers.emplace_back(outTensor.data_ptr<float>() + elementOffset);
+                prefetchPointers.emplace_back(outTensor.template data_ptr<float>() + elementOffset);
                 prefetchSizes.emplace_back(elementCount * sizeof(float));
             }
         }
